activejob/lib/active_job/queue_adapter.rb

```diff
@@ -3,6 +3,15 @@
 require "active_support/core_ext/string/inflections"
 
 module ActiveJob
+  class << self
+    def adapter_name(adapter)
```

Comment:
```suggestion
    def adapter_name(adapter) # :nodoc: 
```

Let keep this private API.